packages/cli/src/types/commands.types.ts

```diff
@@ -36,7 +36,7 @@ interface IExecutionError {
 
 export interface IWorkflowExecutionProgress {
 	workflowId: string;
-	status: ExecutionStatus;
+	status: ExecutionStatus | 'warning';
```

Comment:
Here only used for reporting, not persisted.

Reply:
maybe we should use another type, extend `IWorkflowExecutionProgress` to something like `CliExecutionResult` that could contain `warning` as one of the status, how does it sound? Maybe outside of the scope of this PR.

Reply:
Agree, for now I'll keep this PR about simplifying `ExecutionStatus`. Let's refine types for CLI commands in future.